rust-iml.spec

```
@@ -393,7 +395,7 @@ Group: System Environment/Libraries
 Summary: Consumer of snapshot listing
 License: MIT
 Group: System Environment/Libraries
-Requires: rust-iml-agent-comms
+Requires: rust-iml-agent-comms, rust-iml-timer
```

```suggestion
Requires: rust-iml-agent-comms
```

This package does not have a direct requirement on the timer.